eddymotion/model.py

```diff
@@ -313,40 +374,25 @@ def __init__(self, gtab, S0=None, mask=None, **kwargs):
                 "jac",
             )
         }
-        self._model = DiffusionKurtosisModel(gtab, **kwargs)
-
-    def fit(self, data, **kwargs):
-        """Clean-up permitted args and kwargs, and call model's fit."""
-        self._model = self._model.fit(data[self._mask, ...])
-
-    def predict(self, gradient, **kwargs):
-        """Propagate model parameters and call predict."""
-        predicted = np.squeeze(
-            self._model.predict(
-                _rasb2dipy(gradient),
-                S0=self._S0,
-            )
-        )
-        if predicted.ndim == 3:
-            return predicted
 
-        retval = np.zeros_like(self._mask, dtype="float32")
-        retval[self._mask, ...] = predicted
-        return retval
+        self._model = DiffusionKurtosisModel(gtab, **kwargs)
 
 
 class SparseFascicleModel:
     """
     A wrapper of :obj:`dipy.reconst.sfm.SparseFascicleModel.
     """
 
-    __slots__ = ("_model", "_S0", "_mask", "_solver")
+    __slots__ = ("_model", "_parallel", "_S0", "_mask", "_solver")
```

Comment:
so on so forth...

Reply:
> I would prefer the `omp_nthreads` nomenclature so that this parallelization is not confused with other parallelization happening at a higher level (e.g., SLURM running several instances under the same job or NiPype running nodes in parallel)

I usually think of omp as referring to an openMP style process specifically in a C++ context, but this is fine with me if everyone else is on board.

Reply:
exactly, this is *like* the C++ OMP kind of parallelization

Reply:
```suggestion
    __slots__ = ("_model", "_S0", "_mask", "_solver")
```